resolvers/push_notifications/notifications.js

```diff
@@ -1,95 +1,101 @@
-const express = require("express");
+const express = require('express');
 const router = express.Router();
-const admin = require("firebase-admin");
-var serviceAccount = require('PATH TO SERVICE ACCOUNT JSON KEYS');
+const admin = require('firebase-admin');
+const serviceAccount = require('PATH TO SERVICE ACCOUNT JSON KEYS');
 admin.initializeApp({
-    credential: admin.credential.cert(serviceAccount),
+  credential: admin.credential.cert(serviceAccount),
 });
 router.post('/notify-single', async (req, res) => {
-    var registrationToken=req.body.token;
-    var messageTitle = req.body.title;
-    var messageBody = req.body.message;
-    var message = {
-        notification: {
-          title: messageTitle,
-          body: messageBody
-        },
-        token: registrationToken
-    };
-    admin.messaging().send(message)
-  .then((response) => {
-    res.json({
-        message: "Notification sent!",
-        success: true
+  const {
+    token: registrationToken,
+    title: messageTitle,
+    message: messageBody,
+  } = req.body;
+  const message = {
+    notification: {
+      title: messageTitle,
+      body: messageBody,
+    },
+    token: registrationToken,
+  };
+  admin
+    .messaging()
+    .send(message)
+    .then(() => {
+      res.json({
+        message: 'Notification sent!',
+        success: true,
+      });
     })
-  })
-  .catch((error) => {
-    console.log(error);
-    res.json({
-      message: "Notification not sent:(",
-      success: false
-  })
-  });
+    .catch((error) => {
+      console.log(error);
```

Comment:
Can we remove this console log? As soon as your tests are successful we need all debugging statements to be removed before merging